src/NosCore.GameObject/Networking/ClientSession/ClientSession.cs

```diff
@@ -385,9 +388,9 @@ public string GetMessageFromKey(LanguageKey languageKey)
             return GameLanguage.Instance.GetMessageFromKey(languageKey, Account.Language);
         }
 
-        public async Task HandlePackets(IEnumerable<IPacket> packetConcatenated, IChannelHandlerContext? contex = null)
+        public Task HandlePacketsAsync(IEnumerable<IPacket> packetConcatenated, IChannelHandlerContext? contex = null)
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Use the overloading mechanism instead of the optional parameters.](https://app.codacy.com/manual/NosCoreIO/NosCore/pullRequest?prid=5266673)